pkg/multicluster/link.go

```diff
@@ -368,7 +388,7 @@ func newProbeSpec(obj map[string]interface{}) (ProbeSpec, error) {
 func stringField(obj map[string]interface{}, key string) (string, error) {
 	value, ok := obj[key]
 	if !ok {
-		return "", fmt.Errorf("Field '%s' is missing", key)
+		return "", &ErrFieldMissing{Field: key}
```

Comment:
TIOLI: `key` is passed in as an argument so returning it in the error doesn't tell the caller anything they don't already know.  e.g. at the callsite the caller has both the key they passed in and the `efm.Field`, which is redundant and may be confusing.